src/Stratis.Bitcoin.Features.Consensus/ConsensusFeature.cs

```diff
@@ -256,8 +256,8 @@ public IEnumerable<ConsensusRule> GetRules()
                     new BlockHeaderPowContextualRule(),
 
                     // rules that are inside the method ContextualCheckBlock
-                    new Bip113ActivationRule(),
-                    new Bip34ActivationRule(),
+                    new TransactionLocktimeActivationRule(),
+                    new CoinbaseHeighActivationRule(),
```

Comment:
If you rename it like that, you should make sure that we don't lose information that it is related to those BIP, so please with these lines add  a comment that says that the first one is for BIP 113 and the 2nd one is for 34

Reply:
yeah ok good idea, the BIP information is also present in the rule description but its a good practice to put some explanation next to each rule, agree!